src/main/java/org/folio/circulation/domain/policy/CirculationPolicy.java

```diff
@@ -0,0 +1,24 @@
+package org.folio.circulation.domain.policy;
+
+public abstract class CirculationPolicy {
```

Comment:
@kostyantyn-kh What does adding `circulation` to the name help with? What extra information does it give over `Policy`, as all policies within mod-circulation could be thought of as circulation policies?

Reply:
Kostyantyn has left the project. How do you suggest renaming this class?

Reply:
I don't really understand why the class was renamed from `Policy`. I don't understand what `Circulation` adds to the description.

I know there is another class with the same name. As the module grows I expect that to occur more often as more contexts are described. The existing class, is specific to circulation rules I believe.

Personally, I would rename it back to `Policy`

Reply:
Renamed